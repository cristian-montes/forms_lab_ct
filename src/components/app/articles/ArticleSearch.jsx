import React from "react";

export default function ArticleSearch({ newsName,onHandleNewsNameChange, onHandleSubmit }){
    return(
        <form onSubmit={onHandleSubmit}>
            <label htmlFor="newsName">News Name</label>
            <p> User the search below to look for your Fave News</p>
            <input
                id="newsName"
                name="newsName"
                type="text"
                value={newsName}
                onChange={onHandleNewsNameChange}
            />
            <button aria-label="get-news"> Get Your Fave News</button>
            
        </form>
    )
}