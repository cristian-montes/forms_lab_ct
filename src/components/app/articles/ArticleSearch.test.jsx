import React, { Component } from 'react';
import { render } from '@testing-library/react';
import ArticleSearch from './ArticleSearch';

describe('Search query test', () => {
  it('taking in a query and change on submit', () => {
    const { asFragment } = render(
      <ArticleSearch onSubmit="perro" value="perro" onChange="perro" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});