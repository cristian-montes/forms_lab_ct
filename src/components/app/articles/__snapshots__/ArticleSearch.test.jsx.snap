// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search query test taking in a query and change on submit 1`] = `
<DocumentFragment>
  <form>
    <p>
       User the search below to look for your Fave News
    </p>
    <label
      for="newsName"
    >
      News Name
    </label>
    <input
      id="newsName"
      name="newsName"
      type="text"
      value=""
    />
    <button
      aria-label="get-news"
    >
       Get Your Fave News
    </button>
  </form>
</DocumentFragment>
`;
